cli/lfc/src/main/java/org/lflang/cli/Lfc.java

```diff
@@ -79,6 +79,12 @@ public class Lfc extends CliBase {
       description = "Do not invoke target compiler.")
   private boolean noCompile;
 
+  @Option(
+      names = {"--no-verify"},
```

Comment:
Should it be the default to run verification? What are the performance impacts? I'm thinking it might be better to have it the other way around...

Reply:
I think it only runs if there is a `@property` annotation (?)

Reply:
Yes, that is true. But if you have code with properties, you don't necessarily want to check them every time you compile.

Reply:
That makes sense!